packages/kbn-optimizer/src/worker/webpack.config.ts

```diff
@@ -129,7 +130,24 @@ export function getWebpackConfig(bundle: Bundle, worker: WorkerConfig) {
       },
     ],
 
-    plugins: [new CleanWebpackPlugin(), new DisallowedSyntaxPlugin()],
+    plugins: [
+      new CleanWebpackPlugin(),
+      new DisallowedSyntaxPlugin(),
+      ...(worker.dist
+        ? [
+            new CompressionPlugin({
```

Comment:
There is a `distributableConfig` which is already merged in when `worker.dist` is true